admin/templates/Includes/LeftAndMain_Menu.ss

```diff
@@ -1,36 +1,10 @@
 <div class="cms-menu cms-panel cms-panel-layout west" id="cms-menu" data-layout-type="border">
 	<div class="cms-logo-header north">
```

Comment:
It make more sense to pull this line into the include as sometimes we don't use the north section and it would allow further customisation. Same for the MenuStatus include.